lib/components/component.rb

```diff
@@ -1,5 +1,9 @@
 module Components
   class Component < Element
+    def self.model_name
+      ActiveModel::Name.new(Components::Component)
+    end
```

Comment:
Since `Component` inherits from `Element` it should be enough to define this on `Element`?

Reply:
Technically it would work without redefining the model_name on the `Component` class, but then anonymous instances of the `Component` would have the `Components::Element` model name instead of `Components::Component`. Guess it is more like a flavor call. I would prefer to redefine it, but if you dont I can remove it?

Reply:
Oh you're absolutely right, I didn't see the difference. Let's keep it like this!